recipes/boxing/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Made an upstream PR to include it in the sdist: https://github.com/acifani/boxing/pull/2